src/realm/util/file.hpp

```diff
@@ -736,8 +736,6 @@ class File::Map : private MapBase {
     /// mapped file.
     Map() noexcept;
 
-    ~Map() noexcept;
```

Comment:
This function was completely redundant since it behaves exactly the same as the one you get by default, so I removed it.